src/features/staking/StakingModal.tsx

```diff
@@ -22,7 +23,7 @@ export const StakingModal = () => {
 
   const formReturn = useForm();
   const { getValues } = formReturn;
-  const stakingAmountFormatted = getValues().stakingAmount;
+  const stakingAmount = getValues().stakingAmount * 10 ** 4 || 0;
```

Comment:
Make this a bigint.

`BigInt(getValues().stakingAmount || 0n) ** (10n ** 4n)`

Reply:
The issue I'm facing with this is that when a floating number is passed in, trying to convert it to a `BigInt` causes the app to crash